.github/workflows/README.md

```diff
@@ -37,6 +37,41 @@ operating systems:
   DockerHub. Runs for push to `main` (retags to `latest`) and tags (uses tag
   like `6.28.2`) after build.
 
+## Release Workflow
+
+A separate workflow, `Greengage release`, handles the uploading of Debian packages
+to GitHub releases. It is triggered when a release is published and uses a
+composite action to manage package deployment.
+
+### Key Features
+
+- **Triggers:** `release: [published]` - Runs when a release is published,
+including re-publishing.
+- **Concurrency:** Uses the same concurrency group as the CI workflow
+(`Greengage CI-${{ github.ref }}`) to ensure proper sequencing and prevent race
+conditions.
```

Comment:
In greengage-release.yml we use `Greengage CI-${{ github.ref }}`, but in greengage-ci.yml we use
`${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.event.pull_request.number || github.ref }}`.

Is this correct?

In my tests, the Release workflow [does not wait](https://github.com/hilltracer/greengage/actions/runs/20112883555/job/57714737883) for the image and deb-package build jobs to finish. Could this the reason?